src/main/java/seedu/address/logic/commands/FilterCommand.java

```diff
@@ -125,14 +134,20 @@ private void updateFailedFiltersMessage() {
         }
 
         if (!invalidStatusSet.isEmpty()) {
-            failureMessage.append(MESSAGE_STATUS_ALREADY_BEING_FILTERED).append(invalidStatusSet.stream()
+            failureMessage.append(MESSAGE_EXISTING_STATUS_FILTERED).append(invalidStatusSet.stream()
+                    .map(RsvpStatus::getFilterFormat).collect(Collectors.joining(", "))).append("\n");
+        }
+
+        if (!duplicateStatusSet.isEmpty()) {
+            failureMessage.append(MESSAGE_STATUS_ALREADY_BEING_FILTERED).append(duplicateStatusSet.stream()
                     .map(RsvpStatus::getFilterFormat).collect(Collectors.joining(", "))).append("\n");
         }
```

Comment:
Instead of nesting the 2 blocks, maybe the function can just return straightaway once any of the 2 conditions are not met (ie. isEmpty())